### ISSUE TITLE
Photos are shown in overview that do not tag the user

### ISSUE THREAD

**Issue by wilco375** [csvalpha/amber-ui#875]
Likely related to #871

Navigate to the user's tagged photos, click on a photo that they also uploaded, and go back to the user's tagged photos. Now, photos are listed that the user uploaded but was not tagged in.

**Comment by DrumsnChocolate** [comment]
Yeah related symptoms, likely same cause. I didn't notice this particular symptom because I have never uploaded photos

**Comment by DrumsnChocolate** [comment]
#871 is low prio since it describes incomplete behaviour, but this one is more serious since it is incorrect behaviour
